grafana-plugin/src/containers/OutgoingWebhook2Status/OutgoingWebhook2Status.tsx

```diff
@@ -100,9 +113,7 @@ const OutgoingWebhook2Status = observer((props: OutgoingWebhook2StatusProps) =>
               {data.last_response_log.content && (
                 <VerticalGroup>
                   <Label>Response Body</Label>
-                  <SourceCode showClipboardIconOnly>
-                    {JSON.stringify(data.last_response_log.content, null, 4)}
-                  </SourceCode>
+                  <SourceCode showClipboardIconOnly>{format_response_field(data.last_response_log.content)}</SourceCode>
```

Comment:
FYI, Raphael suggested having response status code and response content at the top of the status log. I guess we can revisit some (easy) UI/UX improvements in a later PR.